components/brave_vpn/brave_vpn_service_desktop.h

```diff
@@ -18,6 +18,7 @@
 #include "brave/components/brave_vpn/brave_vpn_data_types.h"
 #include "brave/components/brave_vpn/brave_vpn_os_connection_api.h"
 #include "brave/components/brave_vpn/brave_vpn_service.h"
+#include "brave/components/skus/browser/skus_service.h"
```

Comment:
you don't need this header, but you do need the mojo header for skus

Reply:
Fixed in https://github.com/brave/brave-core/pull/10847/commits/1a9f60fc958fddf956f9bf2d69fac108b8e7fb09